### PR TITLE
Block_Host_with_Fortinet_Firewall_from_Slack

### PR DESCRIPTION
**Pull request by mhofert-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

workflow1

#### slack

slack_message1
slack_message2

#### Artifact

artifact1
artifact2

### CODE REVIEW

File: workflows/Block_Host_With_Fortinet_Firewall_From_Slack/help.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Version History
+
+* 1.0.0 - Initial workflow
+
+# Links
+
+## References
+
+* [Fortigate](https://www.fortinet.com/products/next-generation-firewall.html)
+* [Slack](https://teams.microsoft.com)
```
Comment (mrinehart-r7) [inline workflows/Block_Host_With_Fortinet_Firewall_From_Slack/help.md:65]: Needs to be fixed

File: workflows/Block_Host_With_Fortinet_Firewall_From_Slack/help.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Description
+
+This workflow blocks or unblocks a host with Fortinet Firewall via Slack command and reports information back to Slack.
+
+Sample Slack Trigger Commands:
+
+`@Rapid7 InsightConnect block-host 198.51.100.100`
+
+`!unblock-host 198.51.100.100`
```
Comment (mrinehart-r7) [inline workflows/Block_Host_With_Fortinet_Firewall_From_Slack/help.md:9]: Looks like this command was skipped by the tool, I only thought one command would be present in each workflow.

File: workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **each Slack step will need the channel name updated to suit your Slack environment!** Edit the input with the preset text of `change_me` in each Slack step in the workflow.
+
+After configuring the Slack steps, activate the workflow in order to trigger it.
+ 
+A optional whitelist can be added to the Fortigate Add Host to be Blocked action. To add this list add IP's or domains in the following format `["198.51.100.100", "example.com", "198.51.100.1"]`
+
+By default this workflow will automatically skip blocking private IP addresses, if you’d like to block these as well, set `Skip RFC 1918` option to false in the Add Host to be Blocked step
```
Comment (jschipp-r7) [inline workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md:37]:
```suggestion
By default this workflow will automatically skip blocking private IP addresses, if you’d like to block these as well, set `Skip RFC 1918` option to false in the "Add Host to be Blocked" step.
```

File: workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **each Slack step will need the channel name updated to suit your Slack environment!** Edit the input with the preset text of `change_me` in each Slack step in the workflow.
+
+After configuring the Slack steps, activate the workflow in order to trigger it.
+ 
+A optional whitelist can be added to the Fortigate Add Host to be Blocked action. To add this list add IP's or domains in the following format `["198.51.100.100", "example.com", "198.51.100.1"]`
```
Comment (jschipp-r7) [inline workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md:35]:
```suggestion
An optional whitelist can be added to the Fortigate Add Host to be Blocked action. To add this list add IP's or domains in the following format `["198.51.100.100", "example.com", "198.51.100.1"]`.
```

File: workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+Once the workflow has been imported, **each Slack step will need the channel name updated to suit your Slack environment!** Edit the input with the preset text of `change_me` in each Slack step in the workflow.
+
+After configuring the Slack steps, activate the workflow in order to trigger it.
+ 
+An optional whitelist can be added to the Fortigate Add Host to be Blocked action. To add this list add IP's or domains in the following format `["198.51.100.100", "example.com", "198.51.100.1"]`.
```
Comment (jschipp-r7) [inline workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md:35]:
```suggestion
An optional whitelist can be added to the Fortigate Add Host to be Blocked action. To add this list add IP's or domains in the following format `["198.51.100.100", "example.com", "198.51.100.1"]` and they will not be blocked.
```

File: workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+# Version History
+
+* 1.0.0 - Initial workflow
+
+# Links
+
+## References
+
+* [Fortigate](https://www.fortinet.com/products/next-generation-firewall.html)
+* [Slack](https://teams.microsoft.com)
```
Comment (jschipp-r7) [inline workflows/Block_Host_with_Fortinet_Firewall_from_Slack/help.md:70]:
```suggestion
* [Rapid7 Slack Chatbot](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
```

**Pull request merged**